example/macos/Runner.xcodeproj/project.pbxproj

```diff
@@ -389,9 +412,13 @@
 				FRAMEWORK_SEARCH_PATHS = (
 					$PROJECT_DIR/Flutter/ephemeral,
 					"$(inherited)",
+					"$(PROJECT_DIR)/Flutter/ephemeral",
```

Comment:
Can you fix this duplication of paths? If this was done via the Xcode UI, then remove the one a couple of lines up.